test/errors/tupleDecls-detailed.good

```diff
@@ -0,0 +1,23 @@
+─── error in tupleDecls.chpl:1 [TupleDeclNotTuple] ───
+  Attempt to use tuple declaration to split a value of non-tuple type 'type UnknownType'.
```

Comment:
Where is this "type UnknownType" coming from?  It doesn't appear to be in the source code?  I'm guessing it's some sort of built-in placeholder, but I think it'd be better if it pretty printed itself out as "non-tuple of unknown type" (say) than to pretend that it's a named type that will mean nothing to the user.

All that said, it seems like the better error here would be to just complain that the source code has no idea what `notKnown` is than to worry about de-tupling it.  I.e., maybe I just forgot a `module use` statement and `notKnown` would have tuple type if I'd not forgotten it?  And it feels like the more pressing error that needs fixing (again, if I'm addressing errors one at a time from top to bottom).

Reply:
It is a built-in placeholder. Since Dyno tries to get as far as possible and report as many errors as it can, even if it can't figure out one thing, it assigns "UnknownType" and moves on. I've added special output to make this say "value of type 'unknown type'", which in my opinion is a good enough compromise for the time being (the sentence structure changes a bit too much otherwise, which requires special logic _inside_ the error message to handle unknown types, which in general means every error message would have to have its own special logic for printing various types, which I find to be a little too much maintenance overhead).